{"name": "test_eval[6*9-42]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 54 == 42\n +  where 54 = eval('6*9')", "trace": "test_input = '6*9', expected = 42\n\n    @pytest.mark.parametrize(\"test_input,expected\", [(\"3+5\", 8), (\"2+4\", 6), (\"6*9\", 42)])\n    def test_eval(test_input, expected):\n>       assert eval(test_input) == expected\nE       AssertionError: assert 54 == 42\nE        +  where 54 = eval('6*9')\n\ntest_pytest\\test_expectation.py:6: AssertionError"}, "parameters": [{"name": "test_input", "value": "'6*9'"}, {"name": "expected", "value": "42"}], "start": 1647928352232, "stop": 1647928352232, "uuid": "99ec2934-ba8d-4790-aaec-d491456c9597", "historyId": "284c92bb3d8fc1d9d2b4384108c552c7", "testCaseId": "ff6166714df71c7f259a948c6e62680c", "fullName": "test_pytest.test_expectation#test_eval", "labels": [{"name": "parentSuite", "value": "test_pytest"}, {"name": "suite", "value": "test_expectation"}, {"name": "host", "value": "IT-L-YANGYL"}, {"name": "thread", "value": "21100-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest.test_expectation"}]}